File: src/main/java/com/example/storage/service/IssueService.java
package com.example.storage.service;

import com.example.storage.controller.dto.EquipmentDto;
import com.example.storage.controller.dto.IssueCreateDto;
import com.example.storage.entity.Issue;

import java.util.List;
import java.util.Map;

public interface IssueService {
    Map<String, List<EquipmentDto>> getAvailableEquipment(String code);

    Issue create(String code, IssueCreateDto issueCreateDto);
}

File: src/main/kotlin/ru/nto/storage/service/IssueService.kt
package ru.nto.storage.service

import ru.nto.storage.controller.dto.EquipmentDto
import ru.nto.storage.controller.dto.IssueCreateDto
import ru.nto.storage.entity.Issue

interface IssueService {
    fun getAvailableEquipment(code: String): Map<String, List<EquipmentDto>>

    fun create(code: String, issueCreateDto: IssueCreateDto): Issue
}
